src/renderer/styles/scroll.css

```diff
@@ -16,4 +16,4 @@
 /* Handle on hover */
 ::-webkit-scrollbar-thumb:hover {
   background: white;
-}
+}
```

Comment:
Your editor is not complying with the formatting rules. Make sure you have prettier installed in VSCode and editorconfig. This way we synchronize the editors and avoid rare changes like these.